go/tools/gazel/generator/BUILD

```diff
@@ -0,0 +1,22 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["generator.go"],
+    visibility = ["//visibility:public"],
+    deps = [
+        "@io_bazel_buildifier//core:go_default_library",
+        "//go/tools/gazel/packages:go_default_library",
+        "//go/tools/gazel/rules:go_default_library",
+        "//go/tools/gazel/testdata:go_default_library",
```

Comment:
I think this dep is unused, and also, shouldn't this fail as testdata is 'testonly = 1' ?